.travis.yml

```diff
@@ -8,15 +8,20 @@ addons:
     packages:
       - inkscape
       - kicad
+      - xvfb
 script:
   - bash keep_alive.sh &
   - export NODE_ENV=production
+  - mkdir ~/.kicad_plugins
+  - pushd ~/.kicad_plugins ; wget https://github.com/openscopeproject/InteractiveHtmlBom/releases/download/v2.3/InteractiveHtmlBom.zip ; unzip InteractiveHtmlBom.zip ; rm InteractiveHtmlBom.zip ; popd
   - ./scripts/plug_versions
   - ./scripts/get_boards production $CACHED_BUILD
   - ./configure production $CACHED_BUILD
   - wget https://github.com/ninja-build/ninja/releases/download/v1.6.0/ninja-linux.zip
   - unzip ninja-linux.zip
   - if  [ "${CACHED_BUILD}" != "cached" ]; then ./ninja clean; fi
+  - Xvfb :0 -screen 0 1024x768x24 > X.log 2>&1 &
+  - export DISPLAY=:0
```

Comment:
Speak of the devil: https://github.com/openscopeproject/InteractiveHtmlBom/pull/157

Reply:
Yeah, this will go away soon, I hope. In the meantime, `Xvfb` is a pretty easy solution.

Reply:
> Yeah, this will go away soon, I hope. In the meantime, `Xvfb` is a pretty easy solution.

And why do you need an X Virtual Server if you are not using the display?

Reply:
I don't think we do. I assume the current version just won't run without this?

Reply:
This is what my pull-request fixes.

Reply:
If you need an X Virtual server you can start it from the docker image, you don't need to patch IBOM.

Reply:
Yep, we are big fans of your pull request over here and waiting for it to be merged  ;) We'll then remove this work-around. 

Also funny that you opened your request just when I was looking at this code here.

Reply:
I don't recommend not using X, wxWidgets has bugs that manifest when there is no wxApp.
The workaround with environment variable will be documented with a warning.

IMO it's no big deal to run fake X server, it's lightweight, but it's up to you. I won't accept/fix bug reports that are not reproduced when running with valid display server.

Reply:
Ok, good to know, thanks very much.

Reply:
I think we should follow @qu1ck's advice and keep `Xvfb`. It's lightweight and is (from what I know) more or less the standard way to handle X-dependent things on Travis. I've used it a lot before for that, without any problems.